README.md

```diff
@@ -36,6 +36,7 @@ Distributed Dataflows in Texera, Zuozhi Wang, Avinash Kumar, Shengquan Ni, Chen
 * (4/2022) Optimizing Machine Learning Inference Queries with Correlative Proxy Models, Zhihui Yang, Zuozhi Wang, Yicong Huang, Yao Lu, Chen Li, X. Sean Wang, to appear in VLDB 2022.
 * (6/2022) Demonstration of Collaborative and Interactive Workflow-Based Data Analytics in Texera, Xiaozhen Liu, Zuozhi Wang, Shengquan Ni, Sadeem Alsudais, Yicong Huang, Avinash Kumar, Chen Li, to appear in VLDB 2022.
 * (6/2022) Demonstration of Accelerating Machine Learning Inference Queries with Correlative Proxy Models, Zhihui Yang, Yicong Huang, Zuozhi Wang, Feng Gao, Yao Lu, Chen Li, X. Sean Wang, to appear in VLDB 2022.
+* (7/2022) Drove: Tracking Execution Results of Workflows on Large Datasets, Sadeem Alsudais, to appear in PhD workshop VLDB 2022.
```

Comment:
"to appear in PhD workshop VLDB 2022." -> "to appear in the PhD workshop at VLDB 2022."